feat: convert policy to use gravitee-doc-gen (#60)
gravitee-io/gravitee-policy-rest-to-soap: .docgen/usage.md

For example, a SOAP API `http(s)://GATEWAY_HOST:GATEWAY_PORT/soap?countryName=France` with the following `{{ .Plugin.ID }}`policy SOAP envelope content:

``` XML
<soap:Envelope xmlns:soap="http://www.w3.org/2003/05/soap-envelope\\\" xmlns:web=\\\"http://www.oorsprong.org/websamples.countryinfo">
   <soap:Header/>
   <soap:Body>
      <web:CountryISOCode>
         <web:sCountryName>{#request.params['countryName']}</web:sCountryName>
      </web:CountryISOCode>
   </soap:Body>
</soap:Envelope>
```
Will give you the ISO country code for ```France```.

## ⚠️ Security Warning: XML Injection Prevention

> **Important**: When using the REST-to-SOAP policy, be aware of potential XML injection vulnerabilities. User input embedded directly into SOAP envelopes without proper escaping can expose your API to security risks.

Reviewer: ```suggestion
**Important**: When you use the REST-to-SOAP policy, you must be aware of potential XML injection vulnerabilities. User input embedded directly into SOAP envelopes without proper escaping might expose your API to security risks.
```
